x-pack/plugins/lists/common/schemas/request/delete_endpoint_list_item_schema.test.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { left } from 'fp-ts/lib/Either';
+import { pipe } from 'fp-ts/lib/pipeable';
+
+import { exactCheck, foldLeftRight, getPaths } from '../../siem_common_deps';
+
+import {
+  DeleteEndpointListItemSchema,
+  deleteEndpointListItemSchema,
+} from './delete_endpoint_list_item_schema';
+import { getDeleteEndpointListItemSchemaMock } from './delete_endpoint_list_item_schema.mock';
+
+describe('delete_exception_list_item_schema', () => {
```

Comment:
Nit - should this be `delete_endpoint_list_item_schema`?

Reply:
Yeah it should. I will fix it in the follow up.